hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/weighted/ConsumerNodeLoadEncoder.java

```diff
@@ -0,0 +1,61 @@
+package pl.allegro.tech.hermes.consumers.supervisor.workload.weighted;
+
+import org.agrona.ExpandableDirectByteBuffer;
+import org.agrona.MutableDirectBuffer;
+import pl.allegro.tech.hermes.api.SubscriptionName;
+import pl.allegro.tech.hermes.consumers.subscription.id.SubscriptionId;
+import pl.allegro.tech.hermes.consumers.subscription.id.SubscriptionIds;
+import pl.allegro.tech.hermes.consumers.supervisor.workload.sbe.stubs.ConsumerLoadEncoder;
+import pl.allegro.tech.hermes.consumers.supervisor.workload.sbe.stubs.ConsumerLoadEncoder.SubscriptionsEncoder;
+import pl.allegro.tech.hermes.consumers.supervisor.workload.sbe.stubs.MessageHeaderEncoder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+class ConsumerNodeLoadEncoder {
+
+    private static final int BUFFER_SIZE_IN_BYTES = 100_000;
```

Comment:
Don't we want to add this as a configuration parameter?

Reply:
I was thinking about it. I know that we do so in other encoders. The only problem I have with this parameter is that it is another one in the long list of parameters we already have. Worse yet, it is unclear how to properly set it up. Maybe in the future we can get rid of them and automatically determine their values. 

Anyway, I think your suggestion is reasonable, so I made it configurable.